src/osm/FactHandler.cpp

```diff
@@ -116,7 +127,15 @@ void osm2rdf::osm::FactHandler<W>::node(const osm2rdf::osm::Node& node) {
 
   _writer->writeTriple(subj, IRI__RDF_TYPE, IRI__OSM_NODE);
 
-  writeBoostGeometry(subj, IRI__GEOSPARQL__HAS_GEOMETRY, node.geom());
+  if (_config.addGeometryObjects) {
```

Comment:
This replaces the current and wrong output mode, maybe the switch should be `useGeometryObjects` or something like that?